build.gradle

```diff
@@ -112,16 +114,21 @@ def afuChild      = fromBaseDir.curry(afuDir)
 def plumeDir        = env["PLUME"] ?: jsr308Child("plume-lib")
 def plumeChild      = fromBaseDir.curry(plumeDir)
 
+def langtoolsChild = fromBaseDir.curry(jsr308_langtools)
+
 //JarsToPackage contains both all members that should be on the classpath for this build
 //and all jars (except for checker-framework-inference.jar) that should be included in
 //the jar produced by the allJars command
 def jarsToPackage = [ //Annotation File Utilities paths        //TODO: AFU Jar has some things that are also in checkers.jar
                         "annotation-file-utilities.jar",
                     ].collect { afuChild(it).getAbsolutePath()      } +
+                    [
+                        "dist/lib/javac.jar",
+                    ].collect { langtoolsChild(it).getAbsolutePath() } +
 
                     [//checker relative paths
-                        "checker/dist/$jreJarName",
-                        "checker/dist/javac.jar",
+                        //"checker/dist/$jreJarName",
```

Comment:
Re-test it: running `checker/bin/javac` would throw an Exception, running `checker/bin-devel/javac` would just give a warning. So as we discussed yesterday, it should be fine with current setting.